### PR TITLE
Add support for Autopep8

### PR DESCRIPTION
**Pull request by Daniel-Atanasov**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -150,7 +150,7 @@ jobs:
       - name: Install Swift dependencies (Linux, uncached)
         if: steps.cache-swift.outputs.cache-hit != 'true' && startsWith(matrix.os, 'ubuntu')
         run: |
-          git clone --branch 0.50500.0 --depth 1 git://github.com/apple/swift-format
+          git clone --branch 0.50500.0 --depth 1 https://github.com/apple/swift-format
```
Comment (Daniel-Atanasov) [inline .github/workflows/test.yml:153]: This fixes tests on ubuntu-latest

File: README.md [hunk 2]
```diff
@@ -335,14 +336,15 @@ Some options are not be available for specific linters:
...
+| autopep8              |     ✅      |  ❌ (py)   |
 | black                 |     ✅      |     ✅     |
 | dotnet_format         |     ✅      |     ✅     |
 | erblint               |     ❌      |  ❌ (erb)  |
 | eslint                |     ✅      |     ✅     |
 | flake8                |     ❌      |     ✅     |
 | gofmt                 |     ✅      |  ❌ (go)   |
 | golint                |     ❌      |  ❌ (go)   |
-| mypy                  |     ✅      |     ✅     |
+| mypy                  |     ❌      |  ❌ (py)   |
```
Comment (Daniel-Atanasov) [inline README.md:347]: Mypy doesn't support either, so just an error in README.md

File: test/linters/projects/autopep8/requirements.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+autopep8>=1.5.0
```
Comment (Daniel-Atanasov) [inline test/linters/projects/autopep8/requirements.txt:1]: autopep8>=1.6.0 causes dependency conflicts with flake8 (updating flake8>=4.0.0 also fixes them, I think)

**Comment by pickfire** [comment]
Hi, can I know when will the next release be done to have these changes?

**Pull request merged**